app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/network/service/IdentityService.kt

```diff
@@ -58,4 +61,14 @@ interface IdentityService {
      * @param refreshToken The refresh token needed to obtain a new token.
      */
     fun refreshTokenSynchronously(refreshToken: String): Result<RefreshTokenResponseJson>
+
+    /**
+     * Send a verification email.
+     */
+    suspend fun sendVerificationEmail(body: SendVerificationEmailRequestJson): Result<ResponseBody?>
```

Comment:
It would be preferred to return the actual type from the API if possible but if it is not, we definitely want to convert it to the correct value before it leaves the service.

Reply:
In this case the service either returns a 204 (no content) or a 200 with a body with a token string, no json object just plain text.